### PR TITLE
Powerfist verb addition

### PR DESCRIPTION
**Pull request by Cowbin**
## About The Pull Request
Adds the verb, "Braziled" to the list used when attacking with the powerfist.

## Why It's Good For The Game
Currently the powerfist only has three verbs used for attacks, all of which are rather bland.  These are, smashed, rammed and power-fisted.  Adding in this extra verb will help to add flavortext to using the powerfist, thus enhancing emersion.  

## Changelog
:cl:
fix: fixed a few things
:cl:

### CODE REVIEW

**Comment by XSlayer300** [comment]
meme

**Comment by Smudged** [comment]
This is kinda a low Rpee PR, but If you made it have a very very very rare chance, it'd be more willing to accept it certainly

File: code/game/objects/items/weapons/misc.dm [hunk 1]
```diff
@@ -41,7 +41,7 @@
 	item_state = "powerfist"
 	flags_equip_slot = ITEM_SLOT_BELT
 	force = 10
-	attack_verb = list("smashed", "rammed", "power-fisted")
+	attack_verb = list("smashed", "rammed", "power-fisted", "Braziled")
```
Comment (Pariah919) [inline code/game/objects/items/weapons/misc.dm:44]:
```suggestion
	attack_verb = list("smashed", "rammed", "power-fisted", "braziled")
```

**Comment by TiviPlus** [comment]
har har

**Pull request closed without merging**